frontend/app/validations/project.js

```diff
@@ -0,0 +1,15 @@
+import {
+  validatePresence,
+  validateLength,
+  validateFormat,
+} from 'ember-changeset-validations/validators';
+
+export default {
+  name: [validatePresence(true), validateLength({ max: 100 })],
+  repo: [
+    validatePresence(true),
+    validateLength({ max: 200 }),
+    validateFormat({ type: 'url' }),
```

Comment:
Customize this to make sure the protocol is in the URL

Reply:
done in 
https://github.com/adfinis/Outdated/pull/14/commits/7572607b01a33f53638e097b5113ac15566e417e